recipes/flex/all/conanfile.py

```diff
@@ -24,41 +27,35 @@ class FlexConan(ConanFile):
         "fPIC": True,
     }
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
     def source(self):
-        tools.get(**self.conan_data["sources"][self.version],
-                  destination=self._source_subfolder, strip_root=True)
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def requirements(self):
         self.requires("m4/1.4.19")
 
     def build_requirements(self):
         self.build_requires("m4/1.4.19")
```

Comment:
```suggestion
        self.tool_requires("m4/1.4.19")
```
Please, prefer tool_requires, as they support new Conan 2.x features 😉

Reply:
My fault. Completely right! Thanks!